Add Tanzu Application Catalog as Notation adopter and blog entry (#371)
notaryproject/notaryproject.dev: content/en/blog/2023/vmware-tanzu-application-catalog-using-notation.md

---
title: VMware Tanzu Application Catalog now uses Notation for signing and verifying OCI artifacts
author:  Beltran Rueda
date: 2023-12-19

> draft: true

Reviewer: ```suggestion
draft: false
```

---

> [VMware Tanzu Application Catalog](https://tanzu.vmware.com/application-catalog), the enterprise edition of [Bitnami Application Catalog](https://bitnami.com/), now leverages [Notation](https://github.com/notaryproject/notation) as a tool for signing and verifying open container initiative (OCI) artifacts (container images, Helm charts, and metadata bundles).

Reviewer 2: ```suggestion
[VMware Tanzu Application Catalog](https://tanzu.vmware.com/application-catalog), the enterprise edition of [Bitnami Application Catalog](https://bitnami.com/), now leverages [Notation](https://github.com/notaryproject/notation) as a tool for signing and verifying open container initiative (OCI) artifacts (e.g. container images, Helm charts, and metadata bundles).
```

Tanzu Application Catalog enables enterprises to build their own private catalog of custom-packaged open source application components that are continuously maintained and verifiably tested for use in production environments. Built by leveraging Bitnami’s expertise in packaging hundreds of open source software applications and delivering them to millions of developers, Tanzu Application Catalog aims to address the open source software needs of enterprises by providing them with customized ready-to-deploy open source applications along with extensive metadata for efficient risk assessment.

With this integration, Notation plays a key role in Tanzu Application Catalog’s mission of making open source software enterprise ready.

## What Tanzu Application Catalog achieves by using Notation

> * **Ensure content integrity**: By signing their OCI artifacts using Notation, Tanzu Application Catalog team can help ensure the integrity of the OCI artifacts they deliver to their customers. Tanzu team uses Notation to signs their OCI artifacts, creating a unique fingerprint for each version of the artifact. Any tampering with the OCI artifact will result in a failed verification, alerting users to potential security threats.

Reviewer: ```suggestion
* **Ensure content integrity**: By signing their OCI artifacts using Notation, Tanzu Application Catalog team can help ensure the integrity of the OCI artifacts they deliver to their customers. Tanzu team uses Notation to sign their OCI artifacts, creating a unique fingerprint for each version of the artifact. Any tampering with the OCI artifact will result in a failed verification, alerting users to potential security threats.
```

Reviewer: Only a small grammar error detected. Others look good.

> * **Ensure content integrity**: By signing their OCI artifacts using Notation, Tanzu Application Catalog team can guarantee the integrity of the OCI artifacts they deliver to their customers. The signatures generated by Notation are based on the content, creating a unique fingerprint for each version of the artifact. Any tampering with the OCI artifact will result in a failed verification, alerting users to potential security threats.

Reviewer 2: ```suggestion
* **Ensure content integrity**: By signing their OCI artifacts using Notation, Tanzu Application Catalog team can help ensure the integrity of the OCI artifacts they deliver to their customers. The signatures generated by Notation are based on the content, creating a unique fingerprint for each version of the artifact. Any tampering with the OCI artifact will result in a failed verification, alerting users to potential security threats.
```

> * **Verify authenticity**: Knowing the source of OCI artifacts is crucial for security and compliance of enterprises. Notary Project signatures provide a way to verify the authenticity of the artifacts by confirming the identity of the signer. This ensures that the applications delivered are built from trusted sources, i.e. Tanzu Application Catalog in this case, reducing the risk of deploying compromised or malicious software.

Reviewer 2: ```suggestion
* **Verify authenticity**: Knowing the source of OCI artifacts is crucial for security and compliance of enterprises. Notation, a client from the Notary Project, enable cryptographic signatures to verify artifact authenticity by validating the cryptographic identity of the signer. This ensures that the applications delivered are built from trusted sources, i.e. Tanzu Application Catalog in this case, reducing the risk of deploying unapproved software
```

> * **Interoperability across tools and platforms**: Notation plays a crucial role in standardizing the representation of signatures. This standardization enables interoperability across different tools and platforms that support the OCI image format without being tied to a specific ecosystem.

Reviewer 2: ```suggestion
* **Interoperability across tools and platforms**: Notation plays a crucial role in standardizing the representation of signatures. This standardization enables interoperability across different tools and platforms that support the OCI image format without being tied to a specific ecosystem.
```
```suggestion
* **Interoperability across tools and platforms**: Notary Project along with its client tool Notation has standardized signature format. This standardization enables interoperability across different tools, registries, container orchestrators, and platforms that support the OCI image format.
```

Thus, Notation, with its standards-based tooling for signing and verifying artifacts, helps Tanzu Application Catalog achieve improved security while delivering compliant open source software artifacts for mission critical production use cases.

> To read more about how Tanzu Application Catalog leverages Notation, check out [this blog](https://tanzu.vmware.com/content/vmware-application-catalog-resources/tanzu-application-catalog-leverages-notation).

Reviewer 3: It would helpful to include a sample policy, which makes use of the VMware trust store (setup at step 2), before it's imported in step 3 in the blog.

Author: thanks @gokarnm, will work on updating the blog post and documentation to add that sample policy.

Author: Blog updated thanks
